### ISSUE TITLE
docs: Contribution docs missing `pod install` step

### ISSUE THREAD

**Issue by vpicone** [storybookjs/react-native#68]
Without running `pod install` in the `ios` directory of the `native` example the code fails with the following error. This could be performed during bootstrap or added to the docs as a final installation step?

```
info Building (using "xcodebuild -workspace native.xcworkspace -configuration Debug -scheme native -destination id=43E2EC9D-DED6-463D-BA0B-E74DD0E8214C -derivedDataPath build/native")
..............
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening native.xcworkspace. Run CLI with --verbose flag for more details.
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace native.xcworkspace -configuration Debug -scheme native -destination id=43E2EC9D-DED6-463D-BA0B-E74DD0E8214C -derivedDataPath build/native

User defaults from command line:
    IDEDerivedDataPathOverride = /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/build/native

note: Using new build system
note: Planning build
note: Using build description from disk
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-nativeTests/Pods-nativeTests.debug.xcconfig: unable to open file (in target "nativeTests" in project "native") (in target 'nativeTests' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-nativeTests/Pods-nativeTests.debug.xcconfig: unable to open file (in target "nativeTests" in project "native") (in target 'nativeTests' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-nativeTests/Pods-nativeTests.debug.xcconfig: unable to open file (in target "nativeTests" in project "native") (in target 'nativeTests' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-nativeTests/Pods-nativeTests.debug.xcconfig: unable to open file (in target "nativeTests" in project "native") (in target 'nativeTests' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-native/Pods-native.debug.xcconfig: unable to open file (in target "native" in project "native") (in target 'native' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-native/Pods-native.debug.xcconfig: unable to open file (in target "native" in project "native") (in target 'native' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-native/Pods-native.debug.xcconfig: unable to open file (in target "native" in project "native") (in target 'native' from project 'native')
error: /Users/vincepicone/Programming/Work/storybook-react-native/examples/native/ios/Pods/Target Support Files/Pods-native/Pods-native.debug.xcconfig: unable to open file (in target "native" in project "native") (in target 'native' from project 'native')
warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the native editor. (in target 'native' from project 'native')


** BUILD FAILED **
```

**Comment by dannyhw** [comment]
These contribution docs seem to be a copy of the storybook core contribution docs and it would be necessary to update them to include any react native specific steps.

**Comment by dannyhw** [comment]
now resolved #98
